src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/HttpRequestManager.cs

```diff
@@ -107,7 +107,9 @@ public async Task<HttpResponseMessage> ExecuteAsync(int retryCount = 3)
                     // For CLI users this will look normal, but translating to a DarcAuthenticationFailureException means it opts in to automated failure logging.
                     if (ex is HttpRequestException && ex.Message.Contains(((int) HttpStatusCode.Unauthorized).ToString()))
                     {
-                        _logger.LogError(ex, "Non-continuable HTTP 401 error encountered while making request");
+                        int queryParamIndex = _requestUri.IndexOf('?');
```

Comment:
Helps make gather-drop logs much nicer, something I encountered making this change.